test-partner/Dockerfile.debug-partner

```diff
@@ -0,0 +1,9 @@
+FROM registry.access.redhat.com/ubi8/ubi:latest
+
+RUN \
+    yum -y install pciutils util-linux net-tools && \
+    yum -y update && \
```

Comment:
I think yum -y update should be placed before yum -y install

Reply:
They're interchangeable, but I can swap them if you think it's better.

Reply:
Swapped in the latest commit